grunt/tasks/misc.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+module.exports = (grunt) => {
+  grunt.registerTask('log', () => {
+    const pkg = grunt.file.readJSON('package.json');
+    grunt.log.writeln(`Deployed: ${pkg.name} (v${pkg.version})`);
+  });
+
+  grunt.registerTask('set_version', (target) => {
+    if (target) {
+      grunt.option('target', target);
+      grunt.log.ok(`Version target set to ${grunt.option('target')}`);
+    } else {
+      target = grunt.config('gitinfo.local.branch.current.name');
+    }
```

Comment:
I think this is actually error prone despite having the same bug in the coffeescript implementation. We should set target to a default value of grunt.config and then we also no longer need the if but just do it.